client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -151,6 +165,46 @@ export const useStocktakeColumns = ({
           },
         },
       ],
+      [
+        'locationName',
+        {
+          getSortValue: row => {
+            if ('lines' in row) {
```

Comment:
Can we just remove sort if it's in 'item' view ? or maybe use helper ?

Reply:
Hmm, we still need to sort if it's in 'item' view though? 🤔